src/styles/catalog.scss

```diff
@@ -0,0 +1,25 @@
+.catalog {
+  display: grid;
+  grid-template-columns: repeat($column-amount, 2);
```

Comment:
The value for `grid-template-columns` seems to be incorrect. According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), you should use `$column-width` for the width of the column, not just `2`. Make sure you define and use the correct variable for the column width.